collector/sys.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2022 Eduardo J. Ortega U.
```

Comment:
You can not copyright assign yourself. This must be `The Prometheus Authors`.

Reply:
Fixed